universum.py

```diff
@@ -80,6 +80,8 @@ def main(*args, **kwargs):
         return run(settings)
     except IncorrectParameterError as e:
         parser.error(e.message)
+    except ImportError as e:
+        print e
```

Comment:
I believe we use "print unicode(e)" to prevent wrong display of any special characters we might encounter in exception (e.g. it somehow might include local path, which we cannot guarantee is ascii)

Reply:
The goal is to have string representation of exception, and the default string type is "unicode". Special characters cannot be handled by just converting using constructor.
Therefore, the proposed change is ok, but the reason is slightly different.